app/components/FormComponents/DatePicker/index.js

```diff
@@ -15,7 +15,7 @@ import { inputType, metaType } from '../../../types'
 
 const StyledDatePicker = styled.div`
   ${MEDIA_QUERIES.TABLET} {
-    max-width: 155px;
+    max-width: auto;
```

Comment:
`auto` isn't valid for `max-width`.  

This also causes the other datepickers to go full width.
![image](https://user-images.githubusercontent.com/1067537/63837911-4a36b780-c974-11e9-8ed8-c1fbce8105a6.png)